docs/electronics/Development_Boards/Arduino.md

```diff
@@ -0,0 +1,170 @@
+
+  
+
+### Explanation
+
+Every Arduino sketch **must** have two particular functions:
+
+  
+
+1.  void setup()
+    1.  The setup() function is called when a sketch starts and will only run once, after each powerup or reset of the Arduino board.
+2.  void loop()
+    1.  This function does precisely what its name suggests, that is loops consecutively, allowing your program to change and respond. Whatever code you write inside loop() will keep running as long as the Arduino is receiving power.
+
+  
+
+Let us examine the [Blink](https://github.com/schacon/blink/blob/master/blink.ino) sketch now, line by line.
+
+  
+
+```cpp
+int led = 13;
+```
+
+  
+
+This line assigns a name to the pin that the LED is attached to, i.e. pin 13.
+
+  
+
+Then we have the setup() function, which runs only once. It includes the following line.
+
+```cpp
+pinMode(led, OUTPUT);
+```
+
+  
+
+This tells the Arduino to configure that pin as an output.
+
+  
+
+Then we have the following loop() function.
+
+```cpp
+void loop()
+{
+  digitalWrite(led, HIGH);  // turn the LED on (HIGH is the voltage level)
+  delay(1000);               // wait for a second
+  digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
+  delay(1000);              // wait for a second
+}
+```
+
+  
+
+The digitalWrite() function tells a pin to either switch on (HIGH, or +5V) or off (LOW, or 0V).  The delay() function tells the Arduino to wait for a specified number of milliseconds.
+
+  
+
+Reading Analog Values
+=====================
+
+  
+
+The following circuit reads the voltage from a potentiometer and sends it via USB to the serial port.
+
+  
+
+Schematic
+---------
+
+  
+
+![](https://lh3.googleusercontent.com/AZzU_IbmAeE8aoIJjLG9o-zziRFjad8DRpkug2cT90-hOpMUmzGK83ftjvlMXgMFVfsD5g9-5Oak0cwrV-8hO2V_xx5ZTeKIN5hOtbAFnaOBFfPrnY03gdokQHgT_OJi_pX4HRwg)
+
+  
+
+Code
+----
+
+Copy the code from [here](https://create.arduino.cc/example/builtin/01.Basics%5CAnalogReadSerial/AnalogReadSerial/preview) and paste it into a new sketch in the Arduino IDE and upload the code to the board. After successful uploading open the serial monitor in the IDE by clicking on its button on top right corner. Trying varying the potentiometer’s knob - you should see the stream of values of the serial monitor change.
+
+  
+
+### Explanation
+
+Whenever the serial port is to be used, it should be initialised with the following line _inside_ void setup(). The 9600 refers to the communication speed in bits-per-second.
+
+```cpp
+Serial.begin(9600);
+```
+
+  
+
+The analogRead function reads the voltage at an analog pin and linearly converts it to a value between 0 and 1023. The Serial.println() function prints a variable to the serial monitor, followed by a newline (using Serial.print() to print data without the newline). The delay(1) is to limit the amount of data printed to the serial monitor.
```

Comment:
Could you add a line at the end pointing to the basics of electronics section (with a link) for more uses of Arduino?